### ISSUE TITLE
Server-side route not working as intended?

### ISSUE THREAD

**Issue by kwhinnery** [TwilioDevEd/hangouts-clone-react#8]
I tried to visit `:3001` to see the app running, but got an error that `res` was not defined ([which is true, the parameter in the function is named `response`](https://github.com/TwilioDevEd/hangouts-clone-react/blob/master/server.js#L54)) - is there another way I am supposed to be viewing the app?

**Comment by captainsafia** [comment]
Hm. Sorry this is poorly documented in the README. The application is run purely using `create-react-apps` front-end tooling. `server.js` isn't responsible for serving any assets when the application is running in development mode, as done when the app is running. The (buggy) lines referenced are design to serve the static production build of the application that is generated by running `npm run build` inside `client`. I removed this for now since I'm not sure if we want to add configuration for production builds in a demo app.

So in summary, you access the application at `localhost:3000`. The `/token` endpoint running in the backend can be accessed using `curl -X POST http://localhost:3001/token`.

Let me know if I need to clarify anything above!
